pkg/kdk/update.go

```diff
@@ -143,3 +223,37 @@ func updateBin(logger logrus.Entry) {
 	}
 
 }
+
+// check if kdk config needs to be updated
+func updateConfigCheck(cfg KdkEnvConfig) (out bool) {
+	if cfg.ConfigFile.AppConfig.ImageTag != latestReleaseVersion ||
+		cfg.ConfigFile.ContainerConfig.Labels["kdk"] != latestReleaseVersion ||
+		cfg.ConfigFile.ContainerConfig.Image != cfg.ImageCoordinates() {
+		return true
+	}
+	return false
+}
+
+// update kdk config
+func updateConfig(cfg *KdkEnvConfig, debug bool, logger logrus.Entry) (err error) {
+	cfg.ConfigFile.AppConfig.ImageTag = latestReleaseVersion
+	cfg.ConfigFile.ContainerConfig.Labels["kdk"] = latestReleaseVersion
+	cfg.ConfigFile.ContainerConfig.Image = cfg.ImageCoordinates()
```

Comment:
Feels kind of odd mutating the "object" from outside like this. It's probably fine though.